docs/contribute/code_review.md

```diff
@@ -0,0 +1,68 @@
+# Code Review
+
+MDC Web team members review every single PR before it is merged into the code
+base. We aim for a quick review process, but without sacrificing good component
+APIs and code maintenance.
+
+## PR Assignments
+
+We currently have six team members, and we normally have bandwidth to respond to
+two PRs each a day (thats twelve PRs total). We do NOT have an SLA to respond
```

Comment:
Either clarify what the acronyms stand for or remove them entirely. Users may not understand what PR or SLA are immediately.